yarn.lock

```diff
@@ -381,10 +381,6 @@ esutils@^2.0.2:
   version "2.0.2"
   resolved "https://registry.yarnpkg.com/esutils/-/esutils-2.0.2.tgz#0abf4f1caa5bcb1f7a9d8acc6dea4faaa04bac9b"
 
-exit@^0.1.2:
```

Comment:
I don't think yarn.lock should change with this patch.

Reply:
reverted this change.